js/plugins/vertexai/src/imagen.ts

```diff
@@ -28,72 +28,140 @@ import { GoogleAuth } from 'google-auth-library';
 import { PluginOptions } from './common/types.js';
 import { PredictClient, predictModel } from './predict.js';
 
+/**
+ * See https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/imagen-api.
+ *
+ * TODO: Remove common config schema extension since Imagen models don't support
+ * most of the common config parameters. Also, add more parameters like sampleCount
+ * from the above reference.
```

Comment:
The todo is fine, but I suggest moving it out of the tsdoc so it doesn't show up in the API refs, and potentially creating and linking to a GH issue.

Reply:
Done